website/docs/components/button/partials/code/how-to-use.md

```diff
@@ -77,6 +77,27 @@ This indicates that the Button should take up the full-width of the parent conta
 <Hds::Button @text="Full width button" @isFullWidth={{true}} />
 ```
 
+### Layout
+
+To change the default `block` layout to `inline`, set `@isLine` to `true`.
+
+In contexts where the Button needs to be <em>inline</em>, to inherit the alignment from a parent, you can use the `@isInline` argument
+
+<!-- #### Default block layout
+
+```handlebars
+<Hds::Button @text="default block layout" />
+<Hds::Button @text="default block layout" />
+``` -->
+
+<!-- #### Inline layout -->
+
```

Comment:
We (generally) don't leave commented code in the documentation (it may cause issues with the parser).

```suggestion
```

Reply:
Sorry that was an oversite. I meant to delete it